docs/source/reference/kubernetes/kubernetes-getting-started.rst

```diff
@@ -324,3 +324,32 @@ FAQs
                   type: Directory
 
   For more details refer to :ref:`config-yaml`.
+
+* **I am using a custom image. How can I speed up the pod startup time?**
+
+  You can pre-install SkyPilot dependencies in your custom image to speed up the pod startup time. Simply add these lines at the end of your Dockerfile:
+
+  .. code-block:: dockerfile
+
+    FROM <your base image>
+
+    # Install system dependencies
+    RUN apt update -y && \
+        apt install git gcc rsync sudo patch openssh-server pciutils fuse unzip socat netcat curl -y && \
```

Comment:
```suggestion
        apt install git gcc rsync sudo patch openssh-server pciutils fuse unzip socat netcat-openbsd curl -y && \
```

netcat is not available on ubuntu:24.04. Need to use openbsd or traditional version.

Reply:
thanks for catching that. Also updating our Dockerfiles to reflect this.